PureBasicDebugger/MemoryViewer.pb

```diff
@@ -3,23 +3,117 @@
 ;  Dual licensed under the GPL and Fantaisie Software licenses.
 ;  See LICENSE and LICENSE-FANTAISIE in the project root for license information.
 ;--------------------------------------------------------------------------------------------
+Global Dim MemoryViewer_Chars.s(31)
 
+#MEMORY_VIEW_TABLE_DATA_DEC = 0 
+#MEMORY_VIEW_TABLE_DATA_HEX = 1 
+#MEMORY_VIEW_TABLE_DATA_OCT = 2 
 
-Global Dim MemoryViewer_Chars.s(31)
+Global MemoryViewTableData.i = MemoryIsHex 
```

Comment:
I should really update the `validate.sh` script part that deals with PB sources and make it fail the build if there is trailing whitespace at EOL (currently checks only for IDE setting in PB files).

I originally intended to do so, but since commits with IDE settings kept coming in I gave priority to that, and postponed the whitespace checks ... then I got sick and didn't manage to catch up. 

Now I'm recovering, so I just need to catch up with some accumulated work that needs to be done, and after that I'll update the script.

Reply:
Done! PR #57 contains an updated script that will now make Travis CI fail the build if any PB source contains trailing whitespace on a non-empty line.

At this link you can see how the files of this PR (now in master) do in fact fail the build and are duly reported:

https://travis-ci.com/tajmone/purebasic/builds/148020124

So, hopefully we shall not see any more whitespaces making it through the Travis checks!

We now need to supply a script to fix whitespace indentation in PB sources, for I've just realized that the PB IDE reformatter doesn't actually remove them. I'm thinking of adding a Bash script that leverages SED to do this (we should expect even Win users to have Bash via Git).